staging/src/slime.io/slime/modules/limiter/controllers/envoy_local_limit.go

```diff
@@ -55,12 +57,14 @@ func generateRouteConfigs(target *microservicev1alpha2.SmartLimitDescriptor_Targ
 				routeName: routeName,
 				vhostName: vhostName,
 				direction: target.Direction,
+				gw:        gw,
 			}
 			rcs = append(rcs, rc)
 		}
 		return rcs
 	}
 	// build inbound by default
+	log.Infof("build default inbound route config base on %s", target)
```

Comment:
The info level is fine if you do not print too many repeated detail logs during normal operation. You can have a think about if you need to change the log level according to the actual situation.